RDF-Pipeline/t/tests/0049_DependsOn_clock_site/setup-files/node/pipeline.ttl

```diff
@@ -5,5 +5,5 @@
 @prefix : <http://localhost/node/> .
 
 :hello a p:FileNode ;
-  p:dependsOn <http://www.time.gov/timezone.cgi?Eastern/d/-5> .
+  p:dependsOn <http://www.time.gov> .
```

Comment:
Does this actually work?   The Last-Modified and ETag headers do not seem to change, though there is a Date header that changes: curl -I 'http://www.time.gov'